jsonforms-editor/src/editor/components/EditorElement.tsx

```diff
@@ -138,6 +144,20 @@ export const EditorElement: React.FC<EditorElementProps> = ({
               >{`(${ruleEffect})`}</Typography>
             </Grid>
           ) : null}
+          {scope ? (
```

Comment:
Wouldn't it make sense to check for `isEditorControl(wrappedElement)` instead for the scope?
You could also do something like this:
`isEditorControl(wrappedElement) && (<Grid...>) `
this way you don't need an else case

Reply:
I updated the test and inlined the check. 
Thanks for reviewing this.

What is your timeline on publishing the lib to npm?

Reply:
Thank you for the fixes. It looks good now.
We plan to publish this library in Q1/21.